### ISSUE TITLE
Many areas are without building mapwithai

### ISSUE THREAD

**Issue by misibacsi** [facebook/Rapid#960]
### Description

Many areas are without building coverage of MapwithAI ("Microsoft Buildings").
Why?

For examaple:
[https://mapwith.ai/rapid#background=FOMI_2007_2010&datasets=fbRoads,msBuildings&disable_features=building_parts,indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r1025072&map=15.00/46.4197/20.3195](url)

I can show you many other areas within Hungary. The roads are OK, they appear more frequently.

### Version

2.0.3

### What browser are you seeing the problem on? What version are you running?

Firefox v114.0

### The OS you're using

win

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid#background=FOMI_2007_2010&datasets=fbRoads,msBuildings&disable_features=building_parts,indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r1025072&map=15.00/46.4197/20.3195

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/114.0

**Comment by misibacsi** [comment]
Other places without building suggestions (not a full list):
All are in Hungary.

https://mapwith.ai/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r1130378&map=16.35/46.63690/18.95019

https://mapwith.ai/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r575234&map=16.00/45.9696/17.9165

https://mapwith.ai/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r575117&map=15.49/45.9904/18.0250

**Comment by tsmock** [comment]
I don't think there are _any_ building suggestions in Europe -- we use Microsoft Buildings, and it doesn't look like they have (yet) made any dataset available for Europe.

See https://github.com/microsoft?q=building-footprint and https://www.microsoft.com/en-us/maps/building-footprints for locations with AI detected buildings.

**Comment by misibacsi** [comment]
Maybe these info are outdated. 
I use Microsoft Buildings AI suggestions for years in Europe (especially in Hungary).

See https://github.com/microsoft/GlobalMLBuildingFootprints/blob/main/images/country-overview.png

and 

https://github.com/microsoft/GlobalMLBuildingFootprints#readme

What I actually edit in Hungary / Europe ahowing Microsoft Buildings AI suggestions :

https://mapwith.ai/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=indoor,landuse,boundaries,rail,pistes,aerialways,power,past_future,others&map=16.32/46.90583/19.67856

**Comment by misibacsi** [comment]
Maybe these info are outdated. 
I use Microsoft Buildings AI suggestions for years in Europe (especially in Hungary).

See https://github.com/microsoft/GlobalMLBuildingFootprints/blob/main/images/country-overview.png

and 

https://github.com/microsoft/GlobalMLBuildingFootprints#readme

What I actually edit in Hungary / Europe ahowing Microsoft Buildings AI suggestions :

[https://mapwith.ai/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=indoor,landuse,boundaries,rail,pistes,aerialways,power,past_future,others&map=16.32/46.90583/19.67856](url)

**Comment by Bonkles** [comment]
I took a quick look at the Microsoft API responses that we get back directly, specifically around the bounding box described by the URL in the bug: https://mapwith.ai/rapid#background=FOMI_2007_2010&datasets=fbRoads,msBuildings&disable_features=building_parts,indoor,landuse,boundaries,pistes,aerialways,power,past_future,others&id=r1025072&map=15.00/46.4197/20.3195

This zoom/lng/lat triplet gets changed to the following request to the MS API: 

GET https://buildingfootprintsapi.azurewebsites.net/api/BuildingFootprints?searchAreaWkt=POLYGON(( 20.297182630191884 46.43047077046194, 20.297182630191884 46.41075087860534, 20.33814140544598 46.41075087860534, 20.33814140544598 46.43047077046194, 20.297182630191884 46.43047077046194))

Which gives a response of `200 OK` and an empty list for a return value.  

So, at least, according to the Building footprints API there are no buildings here. 

Possibilities why this might be: 

https://github.com/microsoft/GlobalMLBuildingFootprints#why-are-some-locations-missing

It sounds like they're aware that there might be missing data.  

Closing this as it does not appear to be a bug with Rapid.
